src/components/User/Users/components/UserOpList.vue

```diff
@@ -26,24 +26,36 @@
         </span>
       </div>
 
-      <button
+      <router-link
         class="app-list__li"
         v-for="item in records"
         :key="item.id"
-        @click="$emit('op-select', item)">
-        <span class="app-list__cell" :title="item.operationType">
+        :to="{ name: 'users.operationDetails',
```

Comment:
Ugly

Reply:
`formatAssetAmount` is useless

Reply:
Could you give me more details about "ugly"